### PR TITLE
Rework acknowlegding calls

### PR DESCRIPTION
**Pull request by MaxNoe**
This does a few major new things (sorry for not splitting it up)

* Add first tests (Yay!)
* Add travis config and activate it
* Use only the webpage for acknowleding calls
* remove calls older than two hours from the `not_acknowledged_calls` so after some time, the primary shifter is called again and not only the backup shifter.

### CODE REVIEW

**Comment by dneise** [comment]
Did you test this already?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+language: python
+python:
+  - "3.5"
+before_install:
+  - export DISPLAY=:99.0
```
Comment (dneise) [inline .travis.yml]: why do we need as display? For matplotlib for creating the QLA plots? In this case, let's rather use another backend and make sure shifthelper works without a display. People running this shifthelper somewhere do not expect it do need a screen, so we should not need one. 
I can open an issue for this, no need to do it right away.

File: shifthelper/checks.py [hunk 2]
```diff
@@ -138,30 +144,36 @@ def all_recent_alerts_acknowledged(checkname):
...
-        return False
+    now = datetime.utcnow()
 
-    if not all_alerts:
-        return False
+    if alerts is None:
+        try:
+            alerts = get_alerts()
+        except RequestException:
+            log.warning('Could not check acknowledged alerts')
```
Comment (dneise) [inline shifthelper/checks.py]: should we log the exception here, in case it ever happens we might be curious why.

**Comment by MaxNoe** [comment]
I will test this tonight.

**Comment by dneise** [comment]
cool then we can merge tomorrow

**Comment by dneise** [comment]
any update here?

**Comment by MaxNoe** [comment]
Same as in #165, I think it worked fine, but you were backup shifter. I would go for merge

**Comment by dneise** [comment]
Ah so both PRs were tested already?
Interesting ... wasnt aware ... pretty confusing

**Comment by MaxNoe** [comment]
Sorry, no. I  got it wrong. This branch was tested not the other one.

**Pull request merged**